wiki/fr/fr-Installation-for-Windows.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: wiki
+title: "Installation sous Windows"
+lang: "fr"
+permalink: "/wiki/Installation-for-Windows"
+---
+
+# Installation sous Windows
+
+Assurez-vous d'avoir déjà lu la page [Premier pas](Getting-Started) avant de commencer.
+
+1. **Téléchargez et installez un pilote ASIO**. Il est recommandé d'utiliser une interface audio ou une carte son avec un pilote ASIO natif. Si vous n'en avez pas (en particulier pour les cartes son internes), installez ce [pilote ASIO gratuit (ASIO4All)](http://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} avant d'installer Jamulus.
+1. **Téléchargez et installez Jamulus** depuis [la page du projet Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer"}. Si vous avez un avertissement, cliquez sur « Plus d'infos » et « Exécuter quand même » pour démarrer l'installation Jamulus.
+1. **Configurez votre interface audio**. Lorsque vous en aurrez terminé, vous devrez configurer votre matériel audio. Regardez comment configurer ASIO4All si vous l'utilisez (défilez vers le bas), et ensuite consultez la section [Configuration du matériel](Hardware-Setup).
+
+Vous pouvez également [compiler vous même le logiciel](Compiling) si vous le souhaitez.
+
+***
+
+## Configuration de ASIO4All
+*ASIO4All n'est pas recommandé si votre interface audio fournit son propre pilote ASIO, car celui-ci pourrait avoir une meilleure latence.*
+
+Tout d'abord, **fermez toutes les applications** (en particulier celles qui pourraient accéder à votre carte son comme votre navigateur) pour minimiser les conflits. Si l'audio ne fonctionne pas « sortie de la boîte », assurez-vous que seules les **entrées/sorties correctes** de son panneau de contrôle **sont activées**.  
```

Comment:
"« sortie de la boîte »" I'm unsure of this because, while "out of the box" is a commonly used working/slogan in English, its literal translation "sortie de la boîte" in French isn't commonly used. What about "Si l'audio ne fonctionne pas directement" ?

Reply:
I hesitated. Hence the quotation marks.

Reply:
> while "out of the box" is a commonly used working/slogan in English, its literal translation "sortie de la boîte" in French isn't commonly used. What about "Si l'audio ne fonctionne pas directement" ?

Exactly this is a point we must clarify and write in the contribution file. Since I think parts of e.g. the german translation suffer from almost wird to word translations since we both weren't sure what to say in German.